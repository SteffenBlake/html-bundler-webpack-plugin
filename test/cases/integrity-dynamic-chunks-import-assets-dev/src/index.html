<!DOCTYPE html>
<html>
<head>
  <title>Dev</title>
  <!-- load source script -->
  <script src="./main.js" defer="defer"></script>
</head>
<body>
  <h1>Hello World!</h1>
  <img src="imac.png">
</body>
</html>