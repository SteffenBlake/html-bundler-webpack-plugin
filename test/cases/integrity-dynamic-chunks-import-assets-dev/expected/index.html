<!DOCTYPE html>
<html>
<head>
  <title>Dev</title>
  <!-- load source script -->
  <link href="main.css" rel="stylesheet" integrity="sha384-P3uRZYJvNVccCqk2iQVGvwDZOYfrwSq1FHv6Ahsf4lBoPPD3ODc0xdEdeom27+CM" crossorigin="anonymous">
<script src="main.6180b06e.js" defer="defer" integrity="sha384-TD4Fhhp30OA4kbftF98uuckZi3mYNIa2M4myC7iTIRMRka7TMy1ULXcwZdkcXvXb" crossorigin="anonymous"></script>
</head>
<body>
  <h1>Hello World!</h1>
  <img src="imac.d12b81b0.png">
</body>
</html>